pkg/scheduler/apis/config/types.go

```diff
@@ -131,3 +140,63 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string
 }
+
+// Plugins include multiple extension points. When specified, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
+// If no plugin should be invoked at an extension point, an empty list must be provided.
+type Plugins struct {
+	// QueueSort is a list of plugins that should be invoked when sorting pods in the scheduling queue.
+	QueueSort []Plugin
+
+	// PreFilter is a list of plugins that should be invoked at "PreFilter" extension point of the scheduling framework.
+	PreFilter []Plugin
+
+	// Filter is a list of plugins that should be invoked when filtering out nodes that cannot run the Pod.
+	Filter []Plugin
+
+	// PostFilter is a list of plugins that should be invoked when a list of nodes that passed the filtering phase.
+	PostFilter []Plugin
+
+	// Score is a list of plugins that should be invoked when ranking nodes that have passed the filtering phase.
+	Score []Plugin
+
+	// NormalizeScore is a list of plugins that should be invoked once per plugin per scheduling cycle.
+	NormalizeScore []Plugin
+
+	// Reserve is a list of plugins that should be invoked when the last step in a scheduling cycle.
+	Reserve []Plugin
+
+	// Permit is a list of plugins that should be invoked when prevent or delay the binding of a Pod.
```

Comment:
Permit is a list of plugins that control binding of a Pod. These plugins can prevent or delay binding of a Pod.